docs/dev-notes/GIT-WORKFLOW.md

```diff
@@ -46,114 +41,164 @@ With all of that explained, lets go over some common scenarios that you will enc
-<br>
-* Run these `git` commands from your computer:
+* If you want to make a new Alpha release, select the `develop` branch. 
+* Want to make a new Beta release? Select the `alpha` branch. 
+* Want to make a new Production release? Select the `beta` branch. 
 
-```bash
-git fetch 
+Click *Run*. After you do this, this will start the sequence of automatically deploying the SDK code for you. 
 
-git switch develop
-git pull 
+Done! You shouldn't have to do anything else at this point to make a release. There is a chance that a failure will occur during the deployment process. The automated deployment process should send messages to the mobile Slack channel when a deployment is successful or failure. You can keep an eye on Slack during the deployment process. If there is a failure, read the message in Slack for help on how to fix the deployment step. 
 
-git checkout -b alpha
-git push origin alpha 
-```
+To help you better understand the automated deployment process and to fix deployment failures, you can read the sections below: 
 
-* Tell the team that you encountered an issue with making an automated release so it can be fixed. 
-</details>
+## The deployment process 
 
-### Want to promote the latest alpha to beta? 
-(There is an `alpha` branch in the project already that you want to promote to beta)
+Read this section to learn about the steps taken to deploy the code base and how to fix each of the steps if there is an error encountered. 
 
-Follow these steps to promote using the recommended automated method:
-* [Click this link][url-promote-action] > Run workflow > For "Use workflow from", select `alpha` > Run. 
+The deployment process goes through these 3 steps:
 
-<details>
-<summary>Did you encounter a problem with the automated release and you need to manually promote? Follow these steps:</summary>
-<br>
-* Run these `git` commands from your computer:
+### 1. Creating or updating git branches for the release.
 
-```bash
-git fetch 
+Remember the green dots discussed in the section [Deployment flow](#Deployment-flow)? We need to perform some git commands in order to merge commits into a release branch, and optionally delete branches after they are no longer needed. 
 
-git switch alpha
-git pull 
+This deployment step is run when you use the *Promote a release* button in GitHub *Actions* tab as discussed in section [Make a new release](#make-a-new-release). 
+
+**Did you encounter a problem with the automated release and you need to manually perform this step? Follow these steps:**
+
+* Run these `git` commands from your computer:
 
-git checkout -b beta
-git push origin beta
 
-git push origin --delete alpha 
-```
+   ```bash
+   git fetch
+   
+   // TODO update these instructions. the command should be printed from the git bot for promote. 
+   
+   git switch develop
+   git pull
+   
+   git checkout -b alpha
+   git push origin alpha 
+   ```
```

Comment:
This is to make a manual Alpha release, correct ? I think it is worthwhile to mention it.